src/framework/MOM_horizontal_regridding.F90

```diff
@@ -764,7 +768,7 @@ subroutine horiz_interp_and_extrap_tracer_fms_id(fms_id,  Time, conversion, G, t
 
   if (.not.spongeDataOngrid) then
     if (is_root_pe()) &
-      call time_interp_external(fms_id, Time, data_in, verbose=.true., turns=turns)
+      call time_interp_external(fms_id, Time, data_in, verbose=(verbosity>2), turns=turns)
```

Comment:
I like and agree with this change, but I would prefer to set the threshold for getting all this output to a somewhat verbose setting, say `verbose=(verbosity>5)`.